src/main/java/seedu/duke/util/PrefixSyntax.java

```diff
@@ -6,20 +6,23 @@
 public class PrefixSyntax {
 
     /* Prefix definitions */
+    public static final String PREFIX_DELIMITER = "/";
+
     public static final String PREFIX_TITLE = "t";
     public static final String PREFIX_TAG = "tag";
+    public static final String PREFIX_INDEX = "i";
     public static final String PREFIX_PIN = "pin";
+    public static final String PREFIX_CONTENT = "c";
+    public static final String PREFIX_LINE = "ln";
+    public static final String PREFIX_SORT = "sort";
     public static final String PREFIX_TIMING = "timing";
     public static final String PREFIX_RECURRING = "rec";
     public static final String PREFIX_REMIND = "rem";
     public static final String PREFIX_STOP_RECURRING = "stop";
-    public static final String PREFIX_END = "/end";
-    public static final String PREFIX_DATETIME = "/d";
-    public static final String PREFIX_INDEX = "i";
-    public static final String PREFIX_CONTENT = "/c";
-    public static final String PREFIX_LINE = "/ln";
-    public static final String PREFIX_DELIMITER = "/";
+    public static final String PREFIX_DATETIME = "d";
     public static final String STRING_SPLIT_DELIMITER = " ";
```

Comment:
Should we sort this by alphabetical order? I mean not a big issue, but if needed, I can do it in my next push? Or if you prefer to sort by usage, then maybe have a line break between each use group?

Reply:
My intent is sort by usage. Noted on the separation.